### PR TITLE
Why it uses name instead of id?

### PR DESCRIPTION
**Pull request by Darkness6030**
If your pull request is **not** translation or serverlist-related, read the list of requirements below and check each box:

- [x] I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [x] I have ensured that my code compiles, if applicable.
- [x] I have ensured that any new features in this PR function correctly in-game, if applicable.

### CODE REVIEW

**Comment by Anuken** [comment]
PRs must be tested.

**Comment by Darkness6030** [comment]
Ok, fine, will test today

**Comment by Darkness6030** [comment]
And yes, it works

File: core/src/mindustry/ui/fragments/PlayerListFragment.java [hunk 1]
```diff
@@ -200,7 +200,7 @@ public void draw(){
 
                 button.button(Icon.hammer, ustyle,
                     () -> ui.showConfirm("@confirm", Core.bundle.format("confirmvotekick",  user.name()),
-                    () -> Call.sendChatMessage("/votekick " + user.name())))
+                    () -> Call.sendChatMessage("/votekick #" + user.id)))
```
Comment (MEEPofFaith) [inline core/src/mindustry/ui/fragments/PlayerListFragment.java:203]: This is only telling the user to type in `/votekick #69`.

Does the command `/votekick #69` actually work though?

~~(I probably shouldn't be asking this, I don't do any sort of server stuff.)~~

Comment (Anuken) [reply core/src/mindustry/ui/fragments/PlayerListFragment.java:203]: I would hope so, considering he said it was tested.

**Comment by Darkness6030** [comment]
@MEEPofFaith just download last BE build, join any server and try to kick any user. It will work

**Comment by Darkness6030** [comment]
I committed this because while player clicks the kick button and sends the packet, the server can theoretically change the target's name (for example rainbow nickname plugin) and the votekick won't happen

**Pull request merged**